src/styles/components/doughnut-chart.scss

```diff
@@ -10,6 +10,7 @@ $border-width: 2px;
   position: relative;
   display: inline-block;
   margin: 0 $global-margin/5;
+  vertical-align: middle;
```

Comment:
For consistency, should we add this to `bubble.scss` too? If so please check in uniprot-website that it doesn't have unwanted side effects. Thanks!

Reply:
Updated